std/string.d

```diff
@@ -1966,10 +1972,16 @@ auto representation(Char)(Char[] s) @safe pure nothrow @nogc
 }
 
 
-/++
-    Capitalize the first character of $(D s) and convert the rest of $(D s)
-    to lowercase.
- +/
+/**
+ * Capitalize the first character of $(D s) and convert the rest of $(D s)
```

Comment:
This only works for characters in the ASCII range, right? If so, you should mention that.